go.mod

```diff
@@ -63,7 +63,6 @@ require (
 	github.com/lithammer/dedent v1.1.0
 	github.com/lpabon/godbc v0.1.1 // indirect
 	github.com/miekg/dns v1.1.35
-	github.com/moby/ipvs v1.0.1
```

Comment:
I would say this is a win :)

Reply:
But apparently we are still vendoring it here: https://github.com/kubernetes/kubernetes/blob/88509404e069137044eeed40cdd38e8f7a1b1df6/go.mod#L366

Because of component-helpers, maybe?